crates/consensus/src/receipt/receipts.rs

```diff
@@ -105,15 +121,20 @@ impl OpTxReceipt for OpDepositReceipt {
 #[derive(Clone, Debug, PartialEq, Eq, Default)]
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 #[cfg_attr(feature = "serde", serde(rename_all = "camelCase"))]
-pub struct OpDepositReceiptWithBloom<T = Log> {
+pub struct OpDepositReceiptWithBloom<T = OpDepositReceipt<Log>> {
```

Comment:
@klkvr do we even need this now, because this should be identical with the alloy receiptWithBloom type @klkvr ?

and we could add an alias instead?

Reply:
yeah, this could be `type OpDepositReceiptWithBloom<T = Log> = ReceiptWithBloom<OpDepositReceipt<T>>` now